js/reference.js

```diff
@@ -52,9 +55,11 @@ var ERMrest = (function(module) {
      * {@link ERMrest.Unauthorized},
      * {@link ERMrest.NotFoundError},
      */
-    module.resolve = function(uri) {
+    function resolve(location) {
```

Comment:
so, yes, back to my comment in parser. obviously you'll just want to fix this with the revised outline i specified above. so the correct parameter is `uri`